files/build_templates/startup_tsa_tsb.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description= STARTUP TSA-TSB SERVICE
+Requires=updategraph.service database.service
+After=updategraph.service database.service
+ConditionPathExists=!/etc/sonic/chassisdb.conf
```

Comment:
what is need for this ?

Reply:
updategraph dependency is to start the tsa-tsb service after config is loaded and chassisdb.conf file check is to start the service only in IMM.